### ISSUE TITLE
Can't open definition of files from other repos

### ISSUE THREAD

**Issue by JJ-Atkinson** [haystackeditor/haystack-editor#30]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 0.19.1
- OS Version:

Steps to Reproduce:

1. Look for an external symbol (e.g. in Java you could try to `goto definition` on Thread/sleep)
2. Usage doesn't open

This seems to happen for all definitions that fall outside the workspace folders.

**Comment by JJ-Atkinson** [comment]
It's been here since 0.18 at least

**Comment by JJ-Atkinson** [comment]
Here's an example error message: 

```
Unable to resolve filesystem provider with relative file path 'jar:file:///home/jarrett/.m2/repository/com/fulcrologic/fulcro-rad/1.6.0/fulcro-rad-1.6.0.jar!/com/fulcrologic/rad/application.cljc'
```

**Comment by AkshaySG14** [comment]
This should be fixed in 0.19.3! Please let me know if it's still broken.

**Comment by JJ-Atkinson** [comment]
Sweet! Can confirm fixed in 0.19.3
